python-threatexchange/threatexchange/extensions/vpdq/tests/test_vpdq_faiss.py

```diff
@@ -37,8 +38,10 @@
 
 if not _DISABLED:
     EXAMPLE_META_DATA = {"example_video"}
```

Comment:
What about this one?

Reply:
I was thinking of giving an example about what the meta_data should be like.

Reply:
Except we don't use meta_data this way anywhere. The Value of T is always FetchedSignalMetadata